bip-0141.mediawiki

```diff
@@ -116,6 +116,16 @@ Blocks are currently limited to 1,000,000 bytes (1MB) total size. We change this
 
 The new rule is ''block weight'' ≤ 4,000,000.
 
+===== Transaction size calculations =====
+
```

Comment:
I mean that they're not mentioned elsewhere in the BIP, so, by standards-speak they are "non-normative" (ie not a part of the spec). Feel free to use english instead of verobse bullshit :p.